realm-transformer/src/main/java/io/realm/transformer/RealmAnalytics.java

```diff
@@ -68,6 +68,7 @@
             + "      \"Anonymized MAC Address\": \"%USER_ID%\",\n"
             + "      \"Anonymized Bundle ID\": \"%APP_ID%\",\n"
             + "      \"Binding\": \"java\",\n"
+            + "      \"Uses Kotlin\": \"%USES_KOTLIN%\",\n"
```

Comment:
Yeah, don’t mind the key name `Language` as opposed to `Languages` as long as we send a List of values I think it’s fine. No need to check if the project uses exclusively Kotlin or Java, as long as we can add either language to the list if we can detect it uses one of them at least once.